docker/ccache.conf

```diff
@@ -0,0 +1,5 @@
+memcached_only = true
+memcached_conf = --SERVER=ccache
+compiler_check = content
+compression = true
+max_size = 1G
```

Comment:
I am not a ccache expert, but we are not adding the conf file equivalent of `CCACHE_CPP2=true` here (seems to be `run_second_cpp` according to the manual)?

Reply:
`run_second_cpp` is enabled by default:
```
# ccache -p|grep run_second_cpp
(default) run_second_cpp = true
```

Reply:
I've added a line in cd5eb4d in case the default changes.